doc/administrator/galaxy/pipelines/mentalist/index.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+search_title: "IRIDA MentaLiST MLST Analsyis"
+description: "Install guide for the MentaLiST pipeline."
+---
+
+MentaLiST MLST Analysis
+=======================
+
+## Step 1: Galaxy Conda Setup
+
+Galaxy makes use of [Conda][conda] to automatically install some dependencies for MentaLiST.  Please verify that the version of Galaxy is >= v16.01 and has been setup to use conda (by modifying the appropriate configuration settings, see [here][galaxy-config] for additional details).
+
+## Step 2: Install Galaxy Tools
+
+Please install all the `mentalist` Galaxy tool by logging into Galaxy, navigating to **Admin > Search and browse tool sheds**, searching for `mentalist` and installing the appropriate **Toolshed Installable Revision**.
```

Comment:
Can you list the **Toolshed Revisions**, similar to https://irida.corefacility.ca/documentation/administrator/galaxy/pipelines/assembly-annotation/.   You will also have to add the `combine_tabular_collection` tool.

Reply:
Awesome.  Looks good.  Thanks.